test/Groups/gsets.jl

```diff
@@ -577,5 +577,5 @@ end
   @test sort(map(length, orbs)) == [1, 1, 1, 3, 3]
   @test all(o -> conductor(sum(collect(o))) == 1, orbs)
   o = orbs[findfirst(o -> length(o) == 3, orbs)]
-  @test [order(permutation(o, x)) for x in gens(u)] == [1, 3]
+  @test sort([order(permutation(o, x)) for x in gens(u)]) == [1, 3]
```

Comment:
```suggestion
  @test issetequal([order(permutation(o, x)) for x in gens(u)], [1, 3])
```
why not use `issetequal`?

Reply:
&lt;snark>my solution requires is 4 bytes shorter&lt;/snark>

I don't care so much as long as we get the pesky temp failures out of the way, and not enough to re-run CI over this, esp since @ThomasBreuer already suggested to replace this with a more robust change (I am all for that, and since Thomas' added these tests in #3833 I fully support him replacing them with even better ones :-).

In the meantime I'll push a fix here for the next test failure...